### DISCUSSION TITLE
CodiceLingua - come valorizzarla

### DISCUSSION THREAD

**Discussion by annamaraspin** [italia/ansc#956] [Supporto servizi cooperativi] [answered]
![image](https://github.com/italia/ansc/assets/127182190/461dbd6a-603d-444b-9f03-e02948c52c46)

il valore evidenziato, codice lingua, è quello della tabella ANPR, corretto?

**Comment by amusarra** [comment, marked as answer]
Buongiorno @annamaraspin 
Confermo che il riferimento per i codici è ANPR (tabella 14 di anpr https://www.anagrafenazionale.interno.it/area-tecnica/tabelle-di-decodifica/).

Cordiali Saluti.
Supporto ANSC.
